testkit/src/main/resources/reference.conf

```
@@ -15,10 +15,13 @@ akka.kafka.testkit {
 # // #testkit-testcontainers-settings
 akka.kafka.testkit.testcontainers {
 
-  # define this to select a different Kafka version by choosing the desired version of Confluent Platform
-  # available Docker images: https://hub.docker.com/r/confluentinc/cp-kafka/tags
+  # define these settings to select a different Kafka/ZooKeeper docker image and version of Confluent Platform
+  # Confluent publishes images on DockerHub: https://hub.docker.com/r/confluentinc/cp-kafka/tags
   # Kafka versions in Confluent Platform: https://docs.confluent.io/current/installation/versions-interoperability.html
-  confluent-platform-version = "6.0.0"
+  confluent-platform-zookeeper-image = "confluentinc/cp-zookeeper"
+  confluent-platform-kafka-image = "confluentinc/cp-kafka"
+  confluent-platform-schema-registry-image = "confluentinc/cp-schema-registry"
+  confluent-platform-version = "6.0.1"
```

```suggestion
  zookeeper-image = "confluentinc/cp-zookeeper"
  zookeeper-version = ${akka.kafka.testkit.testcontainers.confluent-platform-version}
  kafka-image = "confluentinc/cp-kafka"
  kafa-version = ${akka.kafka.testkit.testcontainers.confluent-platform-version}
  kafka-with-schema-registry-image = "confluentinc/cp-schema-registry"
  kafka-with-schema-registry-version = ${akka.kafka.testkit.testcontainers.confluent-platform-version}
  confluent-platform-version = "6.0.1"
```

Would it be beneficial to make this less dependent on the Confluent Platform packaging by individual versions?

---

Good idea.